ui: Say "Node Status" instead of "Node Liveness" in overview

vilterp opened the pull request:
"Status" is a more widely-understood word.

Now:
![screen shot 2018-01-02 at 1 31 58 pm](https://user-images.githubusercontent.com/7341/34495040-5a868e50-efc1-11e7-8891-302983b68f6a.png)

Suggestion by @nstewart in Slack: https://cockroachlabs.slack.com/archives/C2BN46C8L/p1514479614000219

Release note: None

Commit:
ui: Say "Node Status" instead of "Node Liveness" in overview

"Status" is a more widely-understood word.

Fixes #21156

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/21163)
<!-- Reviewable:end -->

couchand commented:
@vilterp can you please also remove the word "Liveness" from the Distributed dashboard?

Closes #21156

Commit:
ui: remove word "Liveness" from Distributed dashboard

Specifically, say "Node Heartbeat Latency" instead of "Node Liveness
Heartbeat Latency".

Release note: None

couchand commented:
LGTM.

Oh, and can you add "Closes #21156" to one of the commit messages please?  Thanks!
